src/ecuacion/PromedioEstudiante.java

```diff
@@ -0,0 +1,81 @@
+package ecuacion;
+import java.util.Scanner;
+import java.time.LocalDate;
+
+public class PromedioEstudiante {
+	
+	public static void main(String[] args) {
+		Scanner scan = new Scanner(System.in);
+		//Ingreso de datos de estudiante
+		System.out.print("Ingrese el Nombre del estudiante: ");
+		String nameStudent = scan.nextLine();
+		System.out.print("Ingrese la Cedula del estudiante: ");
+		String dniStudent = scan.nextLine();
+		System.out.println("Ingrese la fecha de nacimiento del estudiante del estudiante: ");
+		System.out.print("Ingrese el año: ");
+		int yearBorn= scan.nextInt();
+		System.out.print("Ingrese el mes: ");
+		int monthBorn= scan.nextInt();
+		System.out.print("Ingrese el dia: ");
+		int dayBorn= scan.nextInt();
+		
+		//fecha actual en años, meses y dias
+		LocalDate today = LocalDate.now();
+		int yearDate = today.getYear();
+		int monthDate = today.getMonthValue();
+		int dayDate = today.getDayOfMonth();
+		int edad = 0;
+		
+		//Calculo de la edad
+		if(yearDate > yearBorn) {
+			edad = yearDate-yearBorn;
+			if(monthDate < monthBorn) {
+				edad--;
+			}else if(monthDate == monthBorn) {
+				if(dayDate < dayBorn) {
+					edad--;
+				}
+			}
+		}		 
+		int edadActual = edad;
+		//Ingreso de notas
+		System.out.print("Ingrese las notas del estudiante: ");
+		
+		int nota1= scan.nextInt();		
+		
+		if(nota1 >= 0 && nota1<= 20) {
+			int nota2= scan.nextInt();
+			if(nota2 >= 0 && nota2<= 20) {
+				int nota3= scan.nextInt();
+				scan.close();
+				if(nota3 >= 0 && nota3<= 20) {
+					int promedio = (nota1+nota2+nota3)/3;
+					
+					System.out.println(nameStudent+" "+dniStudent);		
```

Comment:
Malardo, no tienes porque estar validando las notas una por una, puedes con un for que se ejecute 3 veces, recibir la nota, validarla y luego meterla en una variable, despues para la segunda nota se volverá a ingresar la nota, se valida de nuevo todo dentro del for, con el mismo if donde se validó la primera nota y lo metes en la misma variable, pero haces que se sumen, por ejemplo, anteriormete meti 10 para la primera nota, y para la segunda nota metí 14, entonces la variable tendrá de valor 24, y asi hasta la tercera nota y esto lo divides entre 3 luego para sacar el promedio, y despues fuera del for, metes el if(promedio>=10) aprobado else no aprobado.

Reply:
Marico porque los estaba haciendo tal cual como lo hicimos en clases pues JAJAJAJAJ

Reply:
Estaba aplicando como si fuese la clase papi

Reply:
Done